lib/net/integration_url_parameter.flow

```diff
@@ -26,10 +28,24 @@ getDomOrUrlParameter(tag : string) -> string {
 getLaunchIdString() -> string {
 	getDomOrUrlParameter("integrationLaunchId");
 }
+
 getIsIntegrationLaunch() -> bool {
 	getLaunchIdString() != "";
 }
 
 getIntegrationLaunchId() -> int {
 	s2i(getLaunchIdString());
 }
+
+getIntegrationLaunchOptions() -> Json {
+	parseJson(getDomOrUrlParameter("integrationOptions"));
+}
+
+canIntegrationShowPWA() {
+	if (getIsIntegrationLaunch()) {
+		options = getIntegrationLaunchOptions();
+		getJsonBoolField(options, "showPWA", false);
+	} else {
+		false;
```

Comment:
Myabe it have to be 'true' when it's not an integration launch?

Reply:
!getIsIntegrationLaunch() || getJsonBoolField(getIntegrationLaunchOptions(), "showPWA", false)